components/skeleton/demo/skeleton-group-nested-test.js

```diff
@@ -0,0 +1,76 @@
+import '../../switch/switch.js';
+import '../../button/button-subtle.js';
+import '../../collapsible-panel/collapsible-panel.js';
+import '../../collapsible-panel/collapsible-panel-group.js';
+import '../../inputs/input-checkbox.js';
+import '../../inputs/input-text.js';
+import '../../list/list.js';
+import '../../list/list-item.js';
+import { css, html, LitElement } from 'lit';
+import { SkeletonGroupMixin } from '../skeleton-group-mixin.js';
+
+class SkeletonTestNestedGroup extends SkeletonGroupMixin(LitElement) {
+	static get properties() {
+		return {
+			_skeletonList: { state: true },
+			_skeletonInput: { state: true },
+		};
+	}
+	static get styles() {
+		return css`
+			.controls {
+				align-items: center;
+				display: flex;
+				gap: 0.6rem;
+				margin-bottom: 0.6rem;
+			}
+			d2l-input-text,
+			d2l-list {
+				margin-bottom: 0.6rem;
+			}
+		`;
+	}
+
+	constructor() {
+		super();
+		this._skeletonList = false;
+		this._skeletonInput = false;
+	}
+
+	render() {
+		return html`
+			<div class="controls">
+				<d2l-switch @click="${this._loadGroup}" ?on="${this._skeleton}" text="group skeleton"></d2l-switch>
+				<d2l-switch @click="${this._loadList}" ?on="${this._skeletonList}" text="d2l-list skeleton"></d2l-switch>
+				<d2l-switch @click="${this._loadInput}" ?on="${this._skeletonInput}" text="d2l-input skeleton"></d2l-switch>
+			</div>
+			<d2l-list ?skeleton="${this._skeletonList}">
+				<d2l-list-item>
+					<d2l-list-item-content>
+						<d2l-input-text label="Label" ?skeleton="${this._skeletonInput}"></d2l-input-text>
+						<d2l-input-checkbox>Item</d2l-input-checkbox>
+						<d2l-input-checkbox>Item</d2l-input-checkbox>
+					</d2l-list-item-content>
+				</d2l-list-item>
+			</d2l-list>
+			<d2l-collapsible-panel-group>
+				<d2l-collapsible-panel panel-title="Title"></d2l-collapsible-panel>
+				<d2l-collapsible-panel panel-title="Title"></d2l-collapsible-panel>
+			</d2l-collapsible-panel-group>
+		`;
+	}
+
+	_loadGroup() {
+		this.skeleton = !this.skeleton;
+	}
```

Comment:
I think I may have found one broken edge case from clicking around wildly. 🙈 Turn on the "`d2l-list` skeleton", then turn on the "group skeleton", then turn off the "`d2l-list` skeleton".  It un-skeletonizes when it shouldn't, because the "group skeleton" switch saw it's `skeleton` was already on and told it to turn itself off when it was toggled.

I actually think this is just a demo problem though, because the switch tries to mess with the value of `skeleton` directly while `SkeletonGroupMixin` is trying to mess with it.  I think this might work?
```suggestion
	_loadGroup() {
		this._skeletonGroup = !this._skeletonGroup;
		this.skeleton = this._skeletonGroup;
	}
```
And add `_skeletonGroup` as another property above, just like `_skeletonList` and `_skeletonInput`